packages/dnb-eufemia/src/extensions/forms/constants/countries.ts

```diff
@@ -2170,7 +2170,7 @@ const countries: Array<CountryType> = [
   },
   {
     i18n: {
-      en: 'United States',
+      en: 'United States of America',
```

Comment:
```suggestion
      en: 'United States',
```

"United States" is the widely recognized short form, while "United States of America" is the official long form used in formal and legal documents, particularly in standards like ISO.

Based on this, it makes sense to opt for "United States" in our context.